tests/global.setup.ts

```
@@ -0,0 +1,14 @@
+import { test as setup } from '@playwright/test'
+import { waitForServerHealth } from './fixtures/server-fixtures'
+
+setup('ensure servers are running', async ({}) => {
```

Biome (1.9.4)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)